buf-ts.gen.yaml

```diff
@@ -1,13 +1,30 @@
-version: v1
+version: v2
+
+inputs:
+  - directory: .
+    paths:
+      - api/proto/teleport/accesslist/
+      - api/proto/teleport/devicetrust/
+      - api/proto/teleport/header/
+      - api/proto/teleport/trait/
+      - api/proto/teleport/userpreferences/
+      - proto/prehog/
+      - proto/teleport/lib/teleterm/
+
 plugins:
-  - name: typescript
-    out: gen/proto/ts
-    opt: eslint_disable,add_pb_suffix,server_grpc1,ts_nocheck
-    path:
+  - local:
       - npm
       - exec
       - --yes
       - --package=@protobuf-ts/plugin@2.9.3
       - --
       - protoc-gen-ts
+    out: gen/proto/ts
+    opt:
+      # the next time we tweak the ts codegen we should put the options in
+      # alphabetical order
```

Comment:
Do it now?  ;)

Reply:
I considered it (as well as bumping the version of `@protobuf-ts/plugin`) but - seeing as the code wouldn't otherwise change - it would just be noise in the PR and in `git blame`.